website/content/tutorials/1-getting-started/_index.md

```diff
@@ -0,0 +1,954 @@
+    </memory>
+  </motherboard>
+</computer>
+{{< /highlight >}}
+{{% /tab %}}
+{{% tab %}}
+{{< highlight yaml >}}
+---
+computer:
+  id: awesomepc1
+  vendor:
+    id: vendor1
+    name: AwesomeComp Incorportated
+    address: 1000 K Street NW Washington, DC 20001
+    website: https://example.com/awesomecomp/awesomepc1/details
+  motherboard:
+    product-name: ISA Corp Magestic Model M-Ultra Motherboard
+    type: atx
+    cpu:
+      product-name: ISA Corp Superchip Model 1 4.7 GHz 4-core Processor
+      architecture: x86-64
+      speed: 4.7 gigahertz
+    ata-socket:
+      product-name: ISA Corp SuperATA Model 2 Socket for SATA Drive
+      type: sata
+    memory-modules:
+    - product-name: Massive Memory Corp Model 3 8GB DDR4-3200 (Module 1)
+      size: 8589934592
+    - product-name: Massive Memory Corp Model 3 8GB DDR4-3200 (Module 2)
+      size: 8589934592
+{{< /highlight >}}
+{{% /tab %}}
+{{< /tabs >}}
+
+With the expressive power of assemblies, flags, and fields, we can specify complex managed objects and control the structure of the intended information model in the resulting data formats.
+
+Our Metaschema-enabled tools can parse and generate the different data formats. We specify flags on the `computer` assembly, but all else we define in the `model` of the `assembly`. And within that model, we can define the motherboard `assembly` inline with its own `flag` and `model`. XML has different syntax and semantics from JSON and YAML that would lead to structural limitations we would need to work around ourselves by manually designing separate data formats without Metaschema. With Metaschema, we can use `group-as` declarations  in our definition, and Metaschema-enabled tools will generate and parse instances for us as instances of the same, despite these limitations, in different data formats automatically.
+
+We define the data types for different Metaschema fields and flags. Our Metaschema-enabled tools can leverage pre-compiled schemas or generate their own to enforce `field` and `flag` values that are valid for their type. For example, our Metaschema-enabled tools should accept a valid URI for the `website` field of the vendor `assembly`, but not any arbitrary string. For `byte-size`, they should only accept positive integer values greater than 0, not a decimal point number or string. Metaschema facilitates consistent enforcement of data typing so we developers do not have to.
+
+We also define the minimum and maximum number of elements for the different assemblies, flags, and field with `min-occurs` and `max-occurs` declarations. In our example, we have an opptional `expansion-card` field in the motherboard `assembly`. Our Metaschema-enabled tools will parse or generate instances as valid with optional fields missing. On the other hand, a motherboard `assembly` missing the CPU `field` should throw errors, as should parsing or generating instances with one that one CPU `field` in the JSON, XML, or YAML format.
```

Comment:
s/opptional/optional

s/with one that one/with more than one

Reply:
```suggestion
We also define the minimum and maximum number of elements for the different assemblies, flags, and field with `min-occurs` and `max-occurs` declarations. In our example, we have an optional `expansion-card` field in the motherboard `assembly`. Our Metaschema-enabled tools will parse or generate instances as valid with optional fields missing. On the other hand, a motherboard `assembly` missing the CPU `field` should throw errors, as should parsing or generating instances with one that one CPU `field` in the JSON, XML, or YAML format.
```